pkg/packagekit/updates.jsx

```diff
@@ -444,11 +444,15 @@ class ApplyUpdates extends React.Component {
                     <strong>{ PK_STATUS_STRINGS[lastAction.status] || PK_STATUS_STRINGS[PK.Enum.STATUS_UPDATE] }</strong>
                     &nbsp;{lastAction.package}
                 </React.Fragment>);
-            logRows = this.state.actions.slice(0, -1).map(action => (
-                <tr key={action.package}>
-                    <th>{PK_STATUS_LOG_STRINGS[action.status] || PK_STATUS_LOG_STRINGS[PK.Enum.STATUS_UPDATE]}</th>
-                    <td>{action.package}</td>
-                </tr>));
+            logRows = [];
+            for (let i = 0; i < this.state.actions.length - 1; i++) {
+                let action = this.state.actions[i];
+                logRows.push(
+                    <tr key={action.package + i}>
```

Comment:
This goes against the idea of `key=`. The key must be an intrinsic value, not an extrinsic one, so that the DOM can still be updated correctly if you insert new things in between. Wouldn't `action.package + action.status` be enough?

Reply:
> This goes against the idea of `key`

Yeah, I know :(  But in this use case we never insert anything in between.

> Wouldn't action.package + action.status be enough?

No, on ubuntu/debian it can print `Updating foo` two or three times in row. I tried to limit these action all together but not successfully.

Reply:
Meh, too bad. So it seems there is no intrinsic unique value. Good enough then.